cloud/deploy/deploy.go

```diff
@@ -151,14 +144,22 @@ func Deploy(path, deploymentID, wsID, pytest, envFile, imageName, deploymentName
 		return err
 	}
 
+	deploymentURL := "cloud." + domain + "/" + deployInfo.workspaceID + "/deployments/" + deployInfo.deploymentID + "/analytics"
+
 	if dags {
-		err = deployDags(path, domain, &deployInfo, client)
+		fmt.Println("Initiating DAGs Deployment for: " + deployInfo.deploymentID)
+		err = deployDags(path, &deployInfo, client)
 		if err != nil {
 			return err
 		}
+
+		fmt.Println("Successfully uploaded DAGs to Astro. Navigate to the Airflow UI to confirm that your deploy was successful. The Airflow UI takes about 1 minute to update." +
```

Comment:
Does it always take 1 minute or could we say something like "may take one minute"

Reply:
Still speaking offline on this. Can update the message in future PR. Going to merge this, since i have more changes coming in based off this.